src/components/Quotes.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState, useEffect } from 'react';
+
+const Quotes = () => {
+  const [quotes, setQuotes] = useState('Loading...');
+
+  useEffect(() => {
+    const getData = async () => {
+      const data = await fetch('https://api.api-ninjas.com/v1/quotes?category=dreams',
+        {
+          method: 'GET',
+          headers: { 'X-Api-Key': 'beSNGjWMHbtj5yQK66y8qw==IhBueBkGLQYQ1tJj' },
+          contentType: 'application/json',
+        }).then((res) => res.json())
+        .then((val) => val)
+        .catch((e) => `Error : ${e}`);
+
+      const val = await data;
+      setQuotes(val);
+    };
```

Comment:
- [ ] while fetching your API, the loading state should be set to true here and when the API response is successful set to false, if there is an error error state should be set to true also